### ISSUE TITLE
Remplir le CRM incubateur

### ISSUE THREAD

**Issue by marinepurson2** [betagouv/animation-communaute#57]

**Comment by marinepurson2** [comment]
On a fait un CRM incubateurs avec @astranchet et il faudrait renseigner dedans les points de contact qu'on a (tech, transverse, etc.) : https://www.notion.so/betagouv-dinum/9b2e0be01d574491a66f6b36a2b35dd7?v=687abd1f17554cc9bd24ccf729291d05

**Comment by marinepurson2** [comment]
Aujourd'hui, il existe tout ça : 
[[Responsables d'incubateurs/de portefeuille](https://docs.google.com/spreadsheets/d/1xnb9N1VF0IxG1Lee9DaNfJKj2UUJLgW_Fc8V292icsY/edit#gid=0)](https://docs.google.com/spreadsheets/d/1xnb9N1VF0IxG1Lee9DaNfJKj2UUJLgW_Fc8V292icsY/edit#gid=0) - d'avril 2023 (donc qui mériterait d'être mis à jour), que

- [[Fanny Blanc [Beta]](https://tchap.gouv.fr/#/user/@fanny.blanc-beta.gouv.fr:agent.dinum.tchap.gouv.fr)](https://tchap.gouv.fr/#/user/@fanny.blanc-beta.gouv.fr:agent.dinum.tchap.gouv.fr) avait fait. On y trouve les "binômes" respospo de portefeuille <-> respopo d'incubateur
- [[Incubateurs et respo accessibilité](https://docs.google.com/spreadsheets/d/1OtFgc92pK5USHdmb_vTdcmIR4vCxsaWMo6udR2nAlg4/edit#gid=1264050583)](https://docs.google.com/spreadsheets/d/1OtFgc92pK5USHdmb_vTdcmIR4vCxsaWMo6udR2nAlg4/edit#gid=1264050583), un CRM perso, qui m'a servi à identifier la personne "accessibilité" de chaque incubateur
- [[Designers Transverses](https://docs.google.com/spreadsheets/d/14bWashbMq61nI-_RO5xctLEWDHUDDbmg95tad1wX7DI/edit#gid=0)](https://docs.google.com/spreadsheets/d/14bWashbMq61nI-_RO5xctLEWDHUDDbmg95tad1wX7DI/edit#gid=0), un (autre) CRM perso, pour identifier les designers transverses dans l'incubateur, s'il y a.
- [[Espace d’échanges](https://docs.google.com/spreadsheets/d/1u2PudRbGkzAkvo14lp1ldODPBn7g7c3m8Lq-OGQbCLg/edit#gid=0)](https://docs.google.com/spreadsheets/d/1u2PudRbGkzAkvo14lp1ldODPBn7g7c3m8Lq-OGQbCLg/edit#gid=0) des incubateurs (Mattermost…)
- [[Paronama des incubateurs](https://grist.incubateur.anct.gouv.fr/o/docs/mxiuQL5Ja1RU/Panorama-des-incubateurs)](https://grist.incubateur.anct.gouv.fr/o/docs/mxiuQL5Ja1RU/Panorama-des-incubateurs) sur Grist

**Comment by fannyblanc** [comment]
Alors, hier Yowa et moi parlions à Amélie Quillet - ils ont développé un super CRM très simple et en interne pour Urban vitaliz. Intéressant d'en parler peut être ?

**Comment by marinepurson2** [comment]
Yes pour l'instant on a monté un truc très simple avec Anne-Sophie pour répondre au besoin d'avoir des listes simples à trouver des référents. Mais ça me va de regarder un CRM plus poussé si on a d'autres besoins identifiés ! On peut en parler au mi-sprint de la semaine prochaine !
